superset/db_engine_specs.py

```diff
@@ -187,26 +187,16 @@ def select_star(cls, my_db, table_name, schema=None, limit=100,
         return sql
 
     @classmethod
-    def modify_url_for_impersonation(cls, url, impersonate_user, username):
+    def get_configuration_for_impersonation(cls, uri, impersonate_user, username):
```

Comment:
Update the configuration dictionary instead of the URL's query dictionary so that we don't have to modify PyHive library.